js/src/components/google-combo-account-card/connect-ads/connect-existing-account.js

```diff
@@ -53,20 +58,19 @@ const ConnectExistingAccount = ( { onCreateClick } ) => {
 			return;
 		}
 
+		clearDetailedErrorBySlot( ERROR_SLOT );
+
 		setLoading( true );
 		try {
 			await connectGoogleAdsAccount();
 			await fetchGoogleAdsAccountStatus();
 			await refetchGoogleAdsAccount();
 		} catch ( error ) {
-			handleApiError(
-				error,
-				null,
-				__(
-					'Unable to connect your Google Ads account. Please try again later.',
-					'google-listings-and-ads'
-				)
-			);
+			receiveDetailedError( ERROR_SLOT, {
```

Comment:
@ankitrox How about we send the `error` object itself instead of just sending the `status` and the `code`? Maybe you can add the custom `title` property to the object that is sent over to `receiveDetailedError`.